src/main/java/org/raml/parser/rule/SchemaRule.java

```diff
@@ -191,6 +191,7 @@ else if (node.getValue().matches("\\w.*"))
     private ScalarNode getGlobalSchemaNode(String key)
     {
         GlobalSchemasRule schemasRule = (GlobalSchemasRule) getRootTupleRule().getRuleByFieldName("schemas");
+        if (schemasRule == null) return null;
```

Comment:
is safe to return null here?

Reply:
yes, when this method is called in line 75 it checks if the returned value is null